### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by ataylor32**

### CODE REVIEW

**Comment by berkerpeksag** [comment]
LGTM. Did you use a tool to find these typos?

**Comment by ataylor32** [comment]
I just ran the comments through a spellchecker

File: tests/context_processors/tests.py [hunk 1]
```diff
@@ -85,5 +85,5 @@ def test_sql_queries(self):
         response = self.client.get(url)
         self.assertContains(response, 'First query list: 0')
         self.assertContains(response, 'Second query list: 1')
-        # Check we have not actually memoized connection.queries
```
Comment (collinanderson) [inline tests/context_processors/tests.py]: actually "memoized" is a thing as silly as it sounds.

**Comment by MarkusH** [comment]
If you could undo the change @collinanderson pointed out, this LGTM

File: django/utils/functional.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def lazy(func, *resultclasses):
     """
     Turns any callable into a lazy evaluated callable. You need to give result
     classes or types -- at least one is needed so that the automatic forcing of
-    the lazy evaluation code is triggered. Results are not memoized; the
+    the lazy evaluation code is triggered. Results are not memorized; the
```
Comment (collinanderson) [inline django/utils/functional.py]: (here too)

**Comment by ataylor32** [comment]
I undid the two "memoized -> memorized" changes and forced a push (to keep history clean).

**Comment by timgraham** [comment]
merged in 039465a6a7a18f48ea77ceadb6949990c0ec92e1, thanks.

**Pull request closed without merging**